src/pykiso/logging_initializer.py

```diff
@@ -317,9 +307,6 @@ def change_logger_class(log_level: str, verbose: bool, logger: str):
         logger_class.__init__ = partialmethod(
             logger_class.__init__, level=LEVELS[log_level], **kwargs_log
```

Comment:
```suggestion
            logger_class.__init__, **kwargs_log
```
This level can be removed else if we change logger class the level will never be internal one for new log